dashboard-app/src/features/dashboard/dataManipulation/logsToAggregatedData.ts

```diff
@@ -26,20 +26,26 @@ export interface AggregatedData {
 
 export const isDataEmpty = (d: AggregatedData) => d.rows.length === 0;
 
-const getIdAndName = (type: string, xData: Params['xData'], value: string | number) => {
+const getIdAndName = (type: TableTypeItem['xIdFromLogs'], xData: Params['xData'], value: string | number) => {
   switch (type) {
     case 'userId':
       const user = xData.find((x) => x.id === Number(value));
-      return { id: user ? user.id : Number(value), name: user ? user.name : 'Deleted User' };
+      return user
+        ? { id: user.id, name: user.name }
+        : { id: Number(value), name: 'Deleted User' };
 
     case 'project':
       const project = xData.find((x) => x.name === value);
-      return { id: project ? project.id : -1, name: project ? project.name : 'General' };
+      return project
+        ? { id: project.id, name: project.name }
+        : { id: -1, name: 'General' };
```

Comment:
Is this not being replaced by the server? Or are you imagining that this PR will go in before those changes?

Reply:
It could potentially go in before the server changes are ready, yeah.